src/linear_algebra/multilinear.lean

```diff
@@ -530,6 +530,21 @@ end semimodule
 
 section
 
+/-- Apply a permutation to the order of the arguments, obtaining another multilinear map.
+
+The naming is derived from `finsupp.dom_congr`, noting that here the permutation applies to the
+domain of the domain. -/
+def dom_dom_congr [add_comm_monoid M₃] [semimodule R M₃]
```

Comment:
Could you add some lemmas describing the multilinear map created here, preferably in terms of `m` and `σ`? I guess you could just use `simps`

Reply:
(Out of curiosity rather than review): does it make any difference putting `simps apply` vs `simps` here? I'd guess no but I could be wrong

Reply:
I think `simps` can end up adding a bunch of extra lemmas that might not we wanted. It probably doesn't matter.